src/plugins/console/server/lib/spec_definitions/json/generated/info.json

```diff
@@ -6,6 +6,6 @@
     "patterns": [
       ""
     ],
-    "documentation": "https://opensearch.org/docs/latest/opensearch/index/"
+    "documentation": "https://opensearch.org/docs/"
```

Comment:
It believe it was previously pointing at the main docs page, which I think is reasonable if we consider pattern ""
<img width="647" alt="image" src="https://user-images.githubusercontent.com/67782303/228841456-ddc8dace-f7e9-4263-b497-b28828d834fc.png">
So I believe updating this link to https://opensearch.org/docs/ will be more appropriate? In this case when no command is written (pattern "") it will point to the index page of opensearch docs.